padertorch/ops/losses/__init__.py

```diff
@@ -1,3 +1,4 @@
 from . import loss
+from . import time_domain_losses
```

Comment:
if we add a time_domain_losses.py we probably have to rename the old loss or/and split it into multiple files

Reply:
I'm not sure what the best naming is here. I first put these losses into `loss`, but the file becomes pretty large then.